File: wallet/src/main/java/com/worldventures/wallet/analytics/new_smartcard/ExistSmartCardAction.java
package com.worldventures.wallet.analytics.new_smartcard;

import com.worldventures.core.service.analytics.AdobeTracker;
import com.worldventures.core.service.analytics.Attribute;
import com.worldventures.janet.analytics.AnalyticsEvent;
import com.worldventures.wallet.analytics.WalletAnalyticsAction;

@AnalyticsEvent(action = "wallet:settings:general:setup new smartcard:existing card detected:card connected",
                trackers = AdobeTracker.TRACKER_KEY)
public class ExistSmartCardAction extends WalletAnalyticsAction {

   @Attribute("unassigncardstep1a") final String unAssignCardStep1a = "1";
}

File: wallet/src/main/java/com/worldventures/wallet/analytics/new_smartcard/ExistSmartCardAction.kt
package com.worldventures.wallet.analytics.new_smartcard

import com.worldventures.janet.analytics.AnalyticsEvent
import com.worldventures.core.service.analytics.Attribute
import com.worldventures.core.service.analytics.AdobeTracker
import com.worldventures.wallet.analytics.WalletAnalyticsAction

@AnalyticsEvent(action = "wallet:settings:general:setup new smartcard:existing card detected:card connected",
      trackers = arrayOf(AdobeTracker.TRACKER_KEY))
class ExistSmartCardAction : WalletAnalyticsAction() {

   @Attribute("unassigncardstep1a") internal val unAssignCardStep1a = "1"
}
